atc/db/resource_type.go

```diff
@@ -322,16 +326,27 @@ func (r *resourceType) CreateBuild(ctx context.Context, manuallyTriggered bool,
 	return build, true, nil
 }
 
+func (r *resourceType) CreateInMemoryBuild(ctx context.Context, plan atc.Plan, seqGen util.SequenceGenerator) (Build, error) {
+	return newRunningInMemoryCheckBuild(r.conn, r.lockFactory, r, plan, NewSpanContext(ctx), seqGen)
```

Comment:
Now that we don't periodically check resource types, I think we can get rid of this - check builds for resource types are only created manually, so they're always stored in the DB

Reply:
Yup, make sense.